package-testing/node-sdk-relay/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
we've got this as well. Pretty sure prettier picks it up in subdirectories.
https://github.com/Eppo-exp/sdk-test-data/blob/main/package-testing/.prettierrc